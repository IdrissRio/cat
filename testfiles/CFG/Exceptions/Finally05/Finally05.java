public class Finally05 {
  static void fun() {
    while (true) {
      System.out.println("InWhile");
      try {
        int f;
        break;
      } finally {
        System.out.println("InFinally");
      }
    }
    System.out.println("AfterWhile");
  }
}