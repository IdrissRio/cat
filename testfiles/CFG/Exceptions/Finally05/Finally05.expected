NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [BooleanLiteral[40]]
NodeName: BooleanLiteral[40] Successors: [TypeAccess[46], TypeAccess[103]]
NodeName: TypeAccess[46] Successors: [VarAccess[47]]
NodeName: VarAccess[47] Successors: [StringLiteral[51]]
NodeName: StringLiteral[51] Successors: [MethodAccess[48]]
NodeName: MethodAccess[48] Successors: [VariableDeclarator[62]]
NodeName: VariableDeclarator[62] Successors: [UncheckedExceptions[66]]
NodeName: UncheckedExceptions[66] Successors: [TypeAccess[75]]
NodeName: TypeAccess[75] Successors: [VarAccess[76]]
NodeName: VarAccess[76] Successors: [StringLiteral[80]]
NodeName: StringLiteral[80] Successors: [MethodAccess[77]]
NodeName: MethodAccess[77] Successors: [TypeAccess[103], Exit[37]]
NodeName: TypeAccess[103] Successors: [VarAccess[104]]
NodeName: VarAccess[104] Successors: [StringLiteral[108]]
NodeName: StringLiteral[108] Successors: [MethodAccess[105]]
NodeName: MethodAccess[105] Successors: [Exit[37]]
NodeName: Exit[37] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[37] Predecessor: [MethodAccess[77], MethodAccess[105]]
NodeName: MethodAccess[77] Predecessor: [StringLiteral[80]]
NodeName: StringLiteral[80] Predecessor: [VarAccess[76]]
NodeName: VarAccess[76] Predecessor: [TypeAccess[75]]
NodeName: TypeAccess[75] Predecessor: [UncheckedExceptions[66]]
NodeName: UncheckedExceptions[66] Predecessor: [VariableDeclarator[62]]
NodeName: VariableDeclarator[62] Predecessor: [MethodAccess[48]]
NodeName: MethodAccess[48] Predecessor: [StringLiteral[51]]
NodeName: StringLiteral[51] Predecessor: [VarAccess[47]]
NodeName: VarAccess[47] Predecessor: [TypeAccess[46]]
NodeName: TypeAccess[46] Predecessor: [BooleanLiteral[40]]
NodeName: BooleanLiteral[40] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: MethodAccess[105] Predecessor: [StringLiteral[108]]
NodeName: StringLiteral[108] Predecessor: [VarAccess[104]]
NodeName: VarAccess[104] Predecessor: [TypeAccess[103]]
NodeName: TypeAccess[103] Predecessor: [BooleanLiteral[40], MethodAccess[77]]
